region/maridia/outer.json

```diff
@@ -1303,12 +1295,31 @@
                   ]
                 },
                 {
-                  "name": "Suitless Frozen Enemies",
+                  "name": "Suitless Frozen Crab",
                   "notable": false,
                   "requires": [
                     "canSuitlessMaridia",
                     "HiJump",
-                    "canUseFrozenEnemies"
+                    "canTrickyUseFrozenEnemies",
```

Comment:
springball jump works in place of HiJump.  But you need a `canStationaryLateralMidAirMorph`.